skip the configured tmpdir if it contains '=', that convention is used only by
mutt, see upstream http://bugs.mutt.org/3324

--- a/smime_keys.pl
+++ b/smime_keys.pl
@@ -946,6 +946,7 @@
 	$option = "notemp" if (not defined($option));
 	if (! $tmpdir and $option eq "temp") {
 		$tmpdir = mutt_Q 'tmpdir';
+                $tmpdir = '/tmp' if ($tmpdir =~ m/=/); # if the tmpdir contains '=', use the default
 		$tmpdir = newfile("$tmpdir/smime");
 		mkdir $tmpdir, 0700 || die "Can't create $tmpdir: $!\n";
 	}
